src/nestegg.c

```diff
@@ -1365,6 +1365,54 @@ ne_find_track_entry(nestegg * ctx, unsigned int track)
   return NULL;
 }
 
+static struct frame *
+ne_alloc_frame()
```

Comment:
It's best to declare functions taking no arguments as `(void)` in regular C.  `()` vs `(void)` can have a different meaning in C vs C++.